client/webserver/site/src/js/settings.js

```diff
@@ -46,6 +53,22 @@ export default class SettingsPage extends BasePage {
     Doc.bind(page.addCert, 'click', () => this.page.certFile.click())
     forms.bind(page.dexAddrForm, page.submitDEXAddr, () => { this.verifyDEX() })
     forms.bind(page.confirmRegForm, page.submitConfirm, () => { this.registerDEX() })
+    forms.bind(page.authorizeAccountExportForm, page.authorizeExportAccountConfirm, () => { this.exportAccount() })
+
+    const exchangesDiv = page.exchanges
+    var exportAccountButton
+    for (const host of Object.keys(app.user.exchanges)) {
+      exportAccountButton = Doc.tmplElement(exchangesDiv, 'exportAccount-' + host)
```

Comment:
Seems like the declaration outside the loop is not necessary.  Can be `const exportAccountButton = ...` inside the loop?